requirements/base.in

```diff
@@ -2,17 +2,9 @@
 -c constraints.txt
 
 Django         # Web application framework
-django-cors-headers
-django-extensions
-django-rest-swagger
-django-waffle
-djangorestframework
-edx-auth-backends
+gunicorn       # HTTP server framework
+
 edx-django-utils
-edx-django-release-util
```

Comment:
Do you know what this is for?

Reply:
No, and the README was pretty vague:

> Release pipeline utilities for edX independently-deployable applications (IDAs) based on Django.

The `__init__.py` said this:

> a collection of Django management commands used for analyzing and manipulating migrations.

...and then it just has a bunch of management commands relating to databases.

Reply:
(So because it was all DB-related, I just removed it.)